docs/dev-guide/modules.md

```diff
@@ -5,18 +5,27 @@ Overview over all modules and dependencies.
 ```mermaid
 flowchart BT
 
+AppModule --> BotModule
+AppModule --> ActivityModule
+AppModule --> CabinModule
+AppModule --> DbModule
+AppModule --> SubscriptionNotifierModule
```

Comment:
Maybe, but for me it made it clear which module is already imported by another module and which one needs to be imported by the AppModule. This way I removed one module from the AppModule import, since it was already imported/initiated by another module. But if we don't care if things are imported multiple times and potentially in places where they are not needed, then we can remove it again I guess?